app/styles/variables/theme.module.scss

```diff
@@ -45,7 +45,7 @@ $comic-issue-signature-aspect-ratio: 380/240;
 
 $creator-banner-aspect-ratio: 1920/900;
 $creator-avatar-aspect-ratio: 500/500;
-$creator-logo-aspect-ratio: 800/450;
+$creator-logo-aspect-ratio: 1000/450;
```

Comment:
Why change this aspect ratio? Logo aspect ratio should be `800/450`, it's the recommended size which Matan has decided on, and we should aim for all creators to respect it if possible. Of course many won't, but we want to push for something like `800/450` rather than `1000/450`